src/components/Identicon.jsx

```diff
@@ -44,15 +44,25 @@ export default class Identicon extends Component {
   }
 
   render() {
-    const {
-      seed = '0x0000000000000000000000000000000000000000',
-      size = 'medium'
-    } = this.props
+
+    const seed = this.props.seed ? this.props.seed : '0x0000000000000000000000000000000000000000'
```

Comment:
should be equivalent with the above (more compact) notation. we should agree on one way to define default values in today's standup